tests/e2e/linalg/fp_to_subbyte.mlir

```diff
@@ -0,0 +1,49 @@
+func.func @f32_to_i4_1d() {
```

Comment:
Total nit: If you do a `git rename` before making changes, git will track it as a rename and is easier to see diff and maintain history.